### DISCUSSION TITLE
Port Forwarding terms

### DISCUSSION THREAD

**Discussion by Twink-Polar** [kingToolbox/WindTerm#56] [General]
Currently WindTerm uses 'Direct' and 'Reverse' terms to describe what is more commonly known as 'Local' and 'Remote' port forwarding.

Would you consider changing the current terms to the more widely known ones?

I apologies if this comes across as me nitpicking, i really like this application and want the SSH component to be as good as it possibly can be.

**Comment by kingToolbox** [comment]
Very nice suggestion, thank you.

"Direct" and "Reverse" are come from libssh document [Forwarding connections](https://api.libssh.org/stable/libssh_tutor_forwarding.html). When I first developed port forwardings, I named these two features according to the documentation. But when I read OpenSSH and putty manuals and documents later, I did find that "Local" and "Remote" were more widely used as you describe.

So I just changed all the terms, and you can see the changes in the next version. I will also revise the related documents, thank you again for your suggestions.

**Comment by kingToolbox** [comment]
It is very necessary to be picky about tools. And I am very happy that you have high standards for WindTerm. We must do this to make excellent tools. I appreciate it very much. :beers:

**Comment by kingToolbox** [comment]
The changes has been released, you can wait for the final stable version or download the new version [2.0.0_prerelease_4](https://github.com/kingToolbox/WindTerm/releases/tag/2.0.0) and check. If nothing else, v2.0.0 will be released today or tomorrow.  

Thank you very much for your concern and suggestions.

**Reply by Twink-Polar** [reply]
Thank you! I'll take the prerelease out for a spin.
